addons/website_event_booth/static/src/scss/website_event_booth.scss

```
@@ -71,3 +71,53 @@ label.o_wbooth_category_unavailable {
 .o_wevent_booth_category {
     background-color: $o-wevent-bg-color-light;
 }
+
+.o_wevent_booth_progress_bar {
+    $wevent-booth-progress-bar-color-1: mix(map-get($theme-colors, 'primary'), $body-bg, 35%);
+    $wevent-booth-progress-bar-color-2: mix(map-get($theme-colors, 'primary'), $body-bg, 45%);
+    height: 48px;
+    li {
+        border: 1px solid $wevent-booth-progress-bar-color-2;
+        border-right-width: 0;
+        padding: 0 24px 0 48px;
+        height: 48px;
+        line-height: 48px;
+        width: 33.333%;
```

```suggestion
        width: calc(100%/3);
```

There's still some rounding non-sense going on which makes the last element look like it has 2 borders at some specific width.
Maybe we can use a css grid but it's arguably less supported and I'm not sure it would solve the issue, leaving it up to you :)